KILTprotocol/kilt-did-driver
# chore: update sdk version

## fixes KILTprotocol/ticket/issues/541

Update to newest SDK version

## How to test:

1. Clone https://github.com/KILTprotocol/universal-resolver
2. build the kilt-did-driver docker image `docker build -t kiltprotocol/kilt-did-driver:1.1.0 .`
3. execute the universal-resolver (inside the `universal-resolver`, `docker-compose up`)
4. wait... wait... The console should be quite... hush little console!
4. query the kilt did: `curl -X GET http://localhost:8080/1.0/identifiers/did:kilt:5GFs8gCumJcZDDWof5ETFqDFEsNwCsVJUj2bX7y4xBLxN5qT`
5 output should be the did doc...

## Checklist:
